FunctionalTableData/Extensions/Array+TableSection.swift

```diff
@@ -29,18 +29,10 @@ extension Array where Element: TableSectionType {
 }
 
 private extension Array where Element: Hashable {
-	func duplicates() -> [Element] {
-		var dups: Set<Element> = []
-		var uniques: Set<Element> = []
-		forEach {
-			if uniques.contains($0) {
-				dups.insert($0)
-			} else {
-				uniques.insert($0)
-			}
-		}
-		return Array(dups)
-	}
+    func duplicates() -> [Element] {
+        let groups = Dictionary(grouping: self, by: { $0 }).filter{ $1.count > 1 }
+        return Array(groups.keys)
+    }
```

Comment:
Oh, just noticed this this is using spaces instead of tabs. Can we update it to be like the rest of the file?

Reply:
@Caiopia After I tried a couple times, I believe this is fixed.